marlowe-runtime/marlowe-tx/Main.hs

```diff
@@ -190,6 +200,14 @@ getOptions = execParser $ info (helper <*> parser) infoMod
       , showDefault
       ]
 
+    contractQueryPortParser = option auto $ mconcat
```

Comment:
Please update `deploy/operables.nix`, `deploy/nomadTasks.nix`, and `nix/dev/compose.nix` accordingly.